bin/pulsar-admin-common.cmd

```diff
@@ -83,14 +83,18 @@ set "OPTS=%OPTS% %PULSAR_EXTRA_OPTS%"
 
 if "%PULSAR_LOG_DIR%" == "" set "PULSAR_LOG_DIR=%PULSAR_HOME%\logs"
 if "%PULSAR_LOG_APPENDER%" == "" set "PULSAR_LOG_APPENDER=RoutingAppender"
-if "%PULSAR_LOG_LEVEL%" == "" set "PULSAR_LOG_LEVEL=info"
-if "%PULSAR_LOG_ROOT_LEVEL%" == "" set "PULSAR_LOG_ROOT_LEVEL=%PULSAR_LOG_LEVEL%"
 if "%PULSAR_ROUTING_APPENDER_DEFAULT%" == "" set "PULSAR_ROUTING_APPENDER_DEFAULT=Console"
 if "%PULSAR_LOG_IMMEDIATE_FLUSH%" == "" set "PULSAR_LOG_IMMEDIATE_FLUSH=false"
 
 set "OPTS=%OPTS% -Dpulsar.log.appender=%PULSAR_LOG_APPENDER%"
 set "OPTS=%OPTS% -Dpulsar.log.dir=%PULSAR_LOG_DIR%"
-set "OPTS=%OPTS% -Dpulsar.log.level=%PULSAR_LOG_LEVEL%"
-set "OPTS=%OPTS% -Dpulsar.log.root.level=%PULSAR_LOG_ROOT_LEVEL%"
+if not "%PULSAR_LOG_LEVEL%" == "" set "OPTS=%OPTS% -Dpulsar.log.level=%PULSAR_LOG_LEVEL%"
+if not "%PULSAR_LOG_ROOT_LEVEL%" == "" (
+    set "OPTS=%OPTS% -Dpulsar.log.root.level=%PULSAR_LOG_ROOT_LEVEL%"
+) else (
+    if not "%PULSAR_LOG_LEVEL%" == "" (
+        set "OPTS=%OPTS% -Dpulsar.log.root.level=%PULSAR_LOG_LEVEL%"
+    )
+)
```

Comment:
Have you had a chance to test on windows that this actually works as expected?
The last time I modified the Windows script, I used this Vagrant configuration on Linux to test changes: https://github.com/lhotari/pulsar-contributor-toolbox/blob/master/vagrant/windows-11/Vagrantfile . It can be used with `vagrant up` and then `vagrant ssh` to ssh into the Windows VM. Password is `vagrant`.

Reply:
I updated the Vagrantfile and added a README for using the Vagrant Windows box: https://github.com/lhotari/pulsar-contributor-toolbox/tree/master/vagrant/windows-11

Reply:
Works well in the windows box. Checked with
```
set PULSAR_LOG_LEVEL=debug
.\bin\pulsar-admin.cmd topics create test
```

Reply:
Use the ps to test this changes, it works fine:
```
$Env:PULSAR_LOG_LEVEL = 'DEBUG'
bin/pulsar-admin tenants list

$Env:PULSAR_LOG_ROOT_LEVEL = 'INFO'
bin/pulsar-admin tenants list
```